### DISCUSSION TITLE
问：节点 node 为什么不是完全按照自己设置的大小，而是会比设置的大小多 12

### DISCUSSION THREAD

**Discussion by laobieyi** [antvis/X6#2751] [Q&A]
比如下图所示：我设置了宽高为36的一个圆， 但是节点是会自动增加一个 12 的距离，导致实际节点会比我们设置的要大。请问有什么解决方法吗？以及导致这个问题的原因是什么？谢谢解答！
![image](https://user-images.githubusercontent.com/110290751/194741582-6a634d24-9fdf-42ac-954a-1bd18622eec5.png)

**Comment by wb-wenbei** [comment]
里面包含了四个port吧

**Reply by laobieyi** [reply]
是的，需要添加 port ，然后现在不知道怎么解决这个间距问题。

**Reply by wb-wenbei** [reply]
可以把port放到圆圈内部

**Reply by laobieyi** [reply]
具体怎么做呢？请教请教

**Reply by wb-wenbei** [reply]
参考下这个例子
https://x6.antv.vision/zh/examples/node/port#port-position-absolute
